client/a8c-for-agencies/README.md

```diff
@@ -0,0 +1,11 @@
+# A8C for Agencies
+
+## Getting Started
+
+You can try out the user-side of Calypso on https://agencies.automattic.com/ or install and run it locally:
```

Comment:
This will currently cause a `no-literal-urls` ESLint error. Let's fix it by making it a proper link:

```suggestion
You can try out the user-side of Calypso on [https://agencies.automattic.com/](https://agencies.automattic.com/) or install and run it locally:
```